_non-cosmos/picasso/assetlist.json

```diff
@@ -77,6 +77,32 @@
           }
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "130",
+          "exponent": 0
+        },
+        {
+          "denom": "usdt",
+          "exponent": 6
+        }
+      ],
+      "base": "130",
+      "name": "Statemine",
+      "display": "usdt",
+      "symbol": "USDT",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "statemine",
+            "base_denom": "130"
+          },
+          "provider": "Statemine"
```

Comment:
I think we've been using something like Kusama Parachain or Kusama Relay for these transfers

Reply:
that was for KSM which came from the Kusama relay chain. Kusama relay chain is connected to Picasso. 
Statemine is also directly connected to Picasso so USDT will transfer directly from Statemine > Picasso